controllers/provider-aws/charts/cloud-provider-config/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Can you move this chart into the `internal` folder?

Reply:
Done.